pyproject.toml

```diff
@@ -43,6 +43,17 @@ Sphinx = "*"
 [tool.poetry.extras]
 docs = ["sphinx"]
 
+[tool.poetry.group.dev.dependencies]
```

Comment:
Do we need both this group and the `[tool.poetry.dev-dependencies]` group?

Reply:
No, I'd like to move the existing one into this, but it breaks things. Thoughts?

Reply:
I'd try `poetry remove --dev <package>` followed by `poetry add --group dev <package>` as the migration process.